host/bcm283x/dma.go

```diff
@@ -508,15 +513,11 @@ func (d *dmaChannel) isAvailable() bool {
 //
 // It doesn't clear the local controlBlock cached values.
 func (d *dmaChannel) reset() {
-	// Make sure nothing is happening.
 	d.cs = dmaReset
 	// Clear bits if needed.
-	d.cs = dmaEnd | dmaInterrupt
-	// Clear values and error bits.
+	// TODO(simokawa): Test if it works as expected.
+	d.cs = dmaInterrupt
 	d.cbAddr = 0
-	d.nextCB = 0
```

Comment:
I think it's better to clear this one too in case it was set.

Reply:
Done.